examples/simple.xml

```diff
@@ -39,14 +102,53 @@
             <NullInst/>
           </config>
           <initialfacilitylist>
+            <entry>
+              <number>1</number>
+              <prototype>OneSource</prototype>
+            </entry>  
+            <entry>
+              <number>1</number>
+              <prototype>OneSink</prototype>
+            </entry>  
             <entry>
               <number>1</number>
               <prototype>OneReactor</prototype>
             </entry>  
+            <!--entry>
+              <number>1</number>
+              <prototype>TwoReactor</prototype>
+            </entry-->  
```

Comment:
I am benchmarking a lot of the functionality against the Cycamore Reactor archetype, so this was an easy way for me to switch between running the simulation with the OpenMCyclus and Cycamore archetypes. I will remove this section because I ended up creating another file that runs with the Cycamore archetype, so this is no longer needed.